### PR TITLE
[O11y][AWS] Migrate AWS Redshift dashboard input controls

### PR DESCRIPTION
**Pull request by kush-elastic**
- Enhancement

## What does this PR do?
- Manually migrate `AWS Redshift dashboard` visualizations to the lens in the current Kibana version `8.8.1` itself. 

- Statistics for `AWS Redshift dashboard` Lens migration:

<html>
<body>
<google-sheets-html-origin>

AWS Redshift Dashboards | Before Migration |   |   |   | After Migration |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | --
  | Control Panel | Search | Lens | Visualization | New Controls | Search | Lens | Visualization
[Metrics AWS] Redshift metrics overview | 1 | 0 | 16 | 0 | 1 | 0 | 16 | 0

</body>
</html>

## Checklist

- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that panels are populated with data.
- [X] I have verified that panels are not distorted after being migrated to the lens.
- [X] I have updated screenshots of the dashboard.
- [X] I have verified that the data counts are matching and panel aggregations are the same as before.

## Author's Checklist

- [X] Migrated panels should be removed from visualization folder.
- [X] Migrated visualizations are populating in current Kibana version 8.8.1 itself.

## Related issues

- Closes https://github.com/elastic/integrations/issues/6862

### CODE REVIEW

**Review by rajvi-elastic — APPROVED** [review]
LGTM!

**Comment by drewdaemon** [comment]
Nice work. Can we swap out those legacy metric visualizations for the normal Lens metric?

**Comment by kush-elastic** [comment]
> Nice work. Can we swap out those legacy metric visualizations for the normal Lens metric?

@drewdaemon, I have created a separate backlog issue (https://github.com/elastic/integrations/issues/6909) to track this and will raise separate PR for converting those visualizations to new metric.

File: packages/aws/kibana/dashboard/aws-81f2c980-e743-11ec-93f6-9b98f71110cd.json [hunk 1]
```diff
@@ -474,29 +392,31 @@
...
+                                "layerType": "data",
+                                "size": "xl",
+                                "textAlign": "center",
+                                "titlePosition": "bottom"
                             }
                         },
                         "title": "",
                         "type": "lens",
-                        "visualizationType": "lnsMetric"
+                        "visualizationType": "lnsLegacyMetric"
```
Comment (mbondyra) [inline packages/aws/kibana/dashboard/aws-81f2c980-e743-11ec-93f6-9b98f71110cd.json:403]: Any reason to use legacy metric over the new metric instead?

**Review by mbondyra — APPROVED** [review]
Looks good to me, seeing that the metrics will be swapped separately 🙆🏼‍♀️

**Review by kaiyan-sheng — APPROVED** [review]

**Comment by kush-elastic** [comment]
The dashboards are exported from 8.7.0 due to export issue from 8.8.x (https://github.com/elastic/elastic-package/issues/1354)

**Pull request merged**